GooglePlacesAutocomplete.js

```diff
@@ -188,7 +189,7 @@ export default class GooglePlacesAutocomplete extends Component {
       }
     }
 
-    navigator.geolocation.getCurrentPosition(
+    Geolocation.getCurrentPosition(
       (position) => {
         if (this.props.nearbyPlacesAPI === 'None') {
           let currentLocation = {
```

Comment:
In the second argument to `getCurrentPosition`, the callback `(error) => {this..., alert...}` change `alert` to `Alert.alert`, as `alert` itself is no longer a global variable in RN and needs to be imported via RN (i.e. `import {Alert} from 'react-native`)